p2p/discover/table.go

```diff
@@ -365,38 +426,109 @@ loop:
 func (tab *Table) doRefresh(done chan struct{}) {
 	defer close(done)
 
+	// Load nodes from the database and insert
+	// them. This should yield a few previously seen nodes that are
+	// (hopefully) still alive.
+	tab.loadSeedNodes(true)
```

Comment:
Previously we only loaded nodes if the refresh failed. Now we load seed nodes always. What's the rationale?